docs/graphkb/index.md

```diff
@@ -0,0 +1,32 @@
+# About
+
+GraphKB is a graph-based implementation of a cancer knowledge base. Unlike other knowledge base projects,
```

Comment:
this sentence is a bit weird. maybe 'graphkb is unique among knowledge base projects in its inclusion of ontology relations....'?

Reply:
yes that's better, changed